system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -195,12 +210,46 @@
               (:revision-id (:meta concept))))
        (is (= "application/vnd.nasa.cmr.umm+json"
               (:format (:meta concept))))
-       (is (= (:ScienceKeywords (:umm concept))
-              [{:Category "EARTH SCIENCE"
+       (is (= [{:Category "EARTH SCIENCE"
                 :Topic "ATMOSPHERE"
                 :Term "AIR QUALITY"
+                :VariableLevel1 "EMISSIONS"}]
+              (:ScienceKeywords (:umm concept)))))))
+
+(deftest bulk-update-update-test
```

Comment:
We should do a find and replace test too and show that only 1 science kw is left in the list.